src/pages/DetailsPage.js

```diff
@@ -40,14 +40,19 @@ const propTypes = {
     /* Onyx Props */
 
     /** The personal details of the person who is logged in */
-    personalDetails: personalDetailsPropType.isRequired,
+    personalDetails: personalDetailsPropType,
 
     /** Route params */
     route: matchType.isRequired,
 
     ...withLocalizePropTypes,
 };
 
+const defaultProps = {
+    // The case when open other profile deep link before login
```

Comment:
So sorry but can we make this comment a bit clearer? Something like:

```suggestion
    // When opening someone else's profile (via deep link) before login, this is empty
```